aten/src/ATen/TensorIterator.h

```diff
@@ -478,13 +486,16 @@ class TORCH_API TensorIteratorConfig final {
   C10_DISABLE_COPY_AND_ASSIGN(TensorIteratorConfig);
 
   /// Construction
+  // Stores input/output Tensors while incrementing the reference count.
+  // Important: the outputs have to be added before the inputs.
   TensorIteratorConfig& add_output(const Tensor& output);
   TensorIteratorConfig& add_input(const Tensor& input);
 
   // Advanced API: stores input/output Tensors without incrementing
-  // the reference count. The caller must ensure that these Tensors
-  // live at least as long as this TensorIteratorConfig and any
-  // TensorIteratorBase built from this TensorIteratorConfig.
+  //   the reference count. The caller must ensure that these Tensors
+  //   live at least as long as this TensorIteratorConfig and any
+  //   TensorIteratorBase built from this TensorIteratorConfig.
```

Comment:
unintentional spaces after // ?

Reply:
fixed, thanks. This was to highlight when a new sentence begins, but didn't fit into the style of the other comments